transaction/Cargo.toml

```diff
@@ -14,7 +14,15 @@ penumbra-tct = { path = "../tct" }
 # Git deps
 decaf377 = { git = "https://github.com/penumbra-zone/decaf377" }
 decaf377-rdsa = { git = "https://github.com/penumbra-zone/decaf377-rdsa" }
-poseidon377 = { git = "https://github.com/penumbra-zone/poseidon377", branch="main" }
+
+# poseidon377 = { git = "https://github.com/penumbra-zone/poseidon377", branch="main" }
+
+# Temporarily pinning poseidon377 until new parameters are audited and the other address-breaking
+# changes are ready to ship.
+poseidon377 = { git = "https://github.com/penumbra-zone/poseidon377", branch="oldparams" }
+ibc = { git = "https://github.com/penumbra-zone/ibc-rs.git", branch = "with-tendermintrs-24" }
+ibc-proto = { git = "https://github.com/penumbra-zone/ibc-rs.git", branch = "with-tendermintrs-24" }
```

Comment:
This will need to be rebased on `main` before merging, so it doesn't touch the Poseidon parameters.